test/feature-benchmark/scenarios.py

```diff
@@ -803,3 +803,63 @@ class KafkaRecovery(KafkaScenario):
 true
 """
     )
+
+
+class KafkaRecoveryBig(Kafka):
```

Comment:
Do we care that this is going to run by default? Similarly, do we want to lower the threshold of significance or perhaps just run it a set number of times and then report the confidence? I think once we fix the persist error this is hitting so it does finish, I'm worried about the overall time of running it a bunch a times in a row to get significance.

Reply:
Yes sorry I meant to change the definition so that it only runs when explicitly requested, and not in Nightly. With respect to the number of iterations , I will make it so that it can be set to a low , predictable value, e.g. `2`

Reply:
That sounds great!

Reply:
I have made it so that it will not run by default. Restricting the number of runs to 2 will need to be a follow-up PR though, I have a refactoring going on that will get me there.